org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/AcquisitionDevice.java

```diff
@@ -179,27 +192,26 @@ public void configure(ScanModel model) throws ScanningException {
 			writers = LevelRunner.createEmptyRunner();
 		}
 		
-		// Create the manager and populate it
-		if (manager!=null) manager.dispose(); // It is allowed to configure more than once.
+		// notify that the device is now ready
+		setDeviceState(DeviceState.READY); 
 		
-		Collection<Object> globalParticipants = ((IScanService)runnableDeviceService).getScanParticipants();
+		// record the time taken to configure the device
+		long after = System.currentTimeMillis();
+		setConfigureTime(after-before);
+	}
 
-		manager = new AnnotationManager(SequencerActivator.getInstance());
+	private AnnotationManager createAnnotationManager(ScanModel model) throws ScanningException {
+		Collection<Object> globalParticipants = ((IScanService)runnableDeviceService).getScanParticipants();
+		AnnotationManager manager = new AnnotationManager(SequencerActivator.getInstance());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Rename "manager" which hides the field declared at line 82. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AHiddenFieldCheck)